modules/distribution/bin/dependencydownloader.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/sh
```

Comment:
can we rename this script name more meaningful?
Eg: adaptivescript.sh, nashron.sh

Reply:
enableadaptive.sh